CONTRIBUTING.md

```diff
@@ -37,6 +37,17 @@ Questions, bug reports, and feature requests can all be submitted as [issues](ht
 
 ## Pull Requests
 
+**NOTE**: We are currently **not** accepting PRs to add new instrumentation
+probes. A new [instrumentation probe API] is being designed. To avoid excessive
+churn and development burden, we will not be accepting new instrumentation
+probes until that API is completed.
```

Comment:
How about using https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts syntax

```suggestion
> [!NOTE]
> We are currently **not** accepting PRs to add new instrumentation
> probes. A new [instrumentation probe API] is being designed. To avoid excessive
> churn and development burden, we will not be accepting new instrumentation
> probes until that API is completed.
```

Reply:
Updated.